Source/WebCore/page/ResizeObservation.cpp

```diff
@@ -105,17 +107,19 @@ FloatSize ResizeObservation::snappedContentBoxSize() const
 std::optional<ResizeObservation::BoxSizes> ResizeObservation::elementSizeChanged() const
 {
     auto currentSizes = computeObservedSizes();
+    if (!currentSizes)
+        return currentSizes;
 
-    LOG_WITH_STREAM(ResizeObserver, stream << "ResizeObservation " << this << " elementSizeChanged - new content box " << currentSizes.contentBoxSize);
+    LOG_WITH_STREAM(ResizeObserver, stream << "ResizeObservation " << this << " elementSizeChanged - new content box " << currentSizes->contentBoxSize);
 
     switch (m_observedBox) {
     case ResizeObserverBoxOptions::BorderBox:
-        if (m_lastObservationSizes.borderBoxLogicalSize != currentSizes.borderBoxLogicalSize)
-            return currentSizes;
+        if (m_lastObservationSizes.borderBoxLogicalSize != currentSizes->borderBoxLogicalSize)
+            return *currentSizes;
         break;
     case ResizeObserverBoxOptions::ContentBox:
-        if (m_lastObservationSizes.contentBoxLogicalSize != currentSizes.contentBoxLogicalSize)
-            return currentSizes;
+        if (m_lastObservationSizes.contentBoxLogicalSize != currentSizes->contentBoxLogicalSize)
+            return *currentSizes;
```

Comment:
The change on this line isn’t needed.

Reply:
I could have sworn it gave compile errors without the change at the time, but indeed now I try again it builds fine.